src/tribler/core/components/restapi/rest/debug_endpoint.py

```diff
@@ -279,9 +281,9 @@ async def get_log(self, request):
         # If the log file exists and return last requested 'max_lines' of log
         try:
             max_lines = int(request.query['max_lines'])
-        except ValueError as e:
-            self.logger.exception(e)
+        except ValueError:
             max_lines = 100
+            self.logger.info(f'Wrong max_lines value. Use the default {max_lines}.')
```

Comment:
1. IMO, it should be an `error` or `warning`, as we will most probably miss the `info` message and do not pay attention to it.
2. Does the message mean that the actual error (missing the expected parameter) is on the GUI side and should also be fixed?
3. If it is not important to fix it on the GUI side, then we can probably use `query.get('max_lines', 100)` instead of catching an error

Reply:
The default value of `query['max_lines']` is `''`

Reply:
Maybe it is worth it to change the default value on the GUI side...

Reply:
I've changed the log level from `info` to `warning` in this case.